tests/unit/checkboxradio/checkboxradio_options.js

```diff
@@ -0,0 +1,180 @@
+-/*
+ * checkboxradio_methods.js
+ */
+
+
+(function($) {
+
+module( "Checkboxradio: checkbox: options" );
+	test( "options: disabled", function() {
+		var checkbox = $( "#checkbox-option-disabled" ),
+			widget;
+		expect( 10 );
+		checkbox.checkboxradio({
+			disabled: true
+		});
+
+		widget = checkbox.checkboxradio( "widget" );
+
+		strictEqual( widget.hasClass( "ui-state-disabled" ), true,
+			"label gets ui-state-disabled when initial option set to true" );
+		strictEqual( checkbox.is( ":disabled" ), true,
```

Comment:
These two assertions get repeated over and over, with just the boolean being toggle, both in the second argument and in the message. Let's abstract these, something like this:

```js
assertEnabled( checkbox );
checkbox.checkboxradio( "option", "disabled", true );
assertDisabled( checkbox );
checkbox.checkboxradio( "option", "disabled", false );
assertEnabled( checkbox );
```